docs/topics/cancellation-and-timeouts.md

```diff
@@ -149,18 +150,19 @@ Replace `while (i < 5)` in the previous example with `while (isActive)` and reru
 
 ```kotlin
 import kotlinx.coroutines.*
+import kotlin.time.Duration.Companion.milliseconds
 
 fun main() = runBlocking {
 //sampleStart
-    val startTime = System.currentTimeMillis()
+    val startTime = markNow()
```

Comment:
How will this compile? Is `markNow` in the scope? Also, `knitCheck` fails, so `./gradlew knit` should be run.

Reply:
Reverted